models/image_classification_mobilenet/README.md

```diff
@@ -29,6 +31,24 @@ python demo.py --input /path/to/image --model v2
 python demo.py --help
 ```
 
+### C++
+
+Install latest OpenCV and CMake >= 3.24.0 to get started with:
+
+```shell
+# A typical and default installation path of OpenCV is /usr/local
+cmake -B build -D OPENCV_INSTALLATION_PATH=/path/to/opencv/installation .
+cmake --build build
+
+# detect on camera input
+./build/opencv_zoo_image_classification_mobilenet
+# detect on an image
+./build/opencv_zoo_image_classification_mobilenet -m=/path/to/model -i=/path/to/image -v
```

Comment:
Do you want to add an option to disable display like 
https://github.com/opencv/opencv_zoo/blob/main/models/text_detection_db/demo.py#L54-L55

Reply:
Sorry I work on three git branch and I must remove rcnn files

Reply:
I think it is better to add it, because the headless version or server version of opencv cannot use GUI. Also keeps the c++ version and python version of the demo consistent.